### ISSUE TITLE
Show skipped filenames as "skipped" in yellow

### ISSUE THREAD

**Issue by ljharb** [rogeliog/jest-runner-eslint#9]
I'd like to see, alongside green PASS and red FAIL, a yellow SKIP when a file is eslintignored (to match up with the nice new skip count).

It'd also be nice if there was an option to suppress reporting these skipped tests (but showing them by default).

**Comment by rogeliog** [comment]
I'm not sure if Jest's reporter provides this feature, I think it doesn't but I might be wrong... If this is the case, then it would be harder to implement this because it might require changes in facebook/jest

**Comment by rogeliog** [comment]
One thing that might help to mitigate the issue is adding the files/globs to [testPathIgnorePatterns](http://facebook.github.io/jest/docs/en/configuration.html#testpathignorepatterns-array-string)

**Comment by ljharb** [comment]
I don't want to duplicate the contents of `.eslintignore` tho - it totally would be nice to have the runner count up the number of ignored files, add that as a skip count (and explicit SKIP entries), and then use them in testPathIgnorePatterns to make the eslint run *even* faster :-D

**Comment by rogeliog** [comment]
I agree, this would be great!

@cpojer there is no way that this could be achieved in the current version of Jest right?

**Comment by ljharb** [comment]
Don't pending tests show up in yellow in jest? That's what I'd assume ignored files would show up as.

**Comment by cpojer** [comment]
Yeah, I think this should work if in the eslint runner you report all tests within a file as skipped.

**Comment by rogeliog** [comment]
Hmmm, I think I'm missing something sorry 😞

Isn't it that it could either show this when skipping all the tests
<img width="484" alt="screen shot 2017-09-12 at 9 59 28 am" src="https://user-images.githubusercontent.com/574806/30338674-3fc77ee4-97a1-11e7-89ce-f94d4de799c8.png">

or this when skipping the test suite
<img width="263" alt="screen shot 2017-09-12 at 10 01 36 am" src="https://user-images.githubusercontent.com/574806/30338720-667761e4-97a1-11e7-8103-14e0defe0bd6.png">

**Comment by ljharb** [comment]
In the case of the eslint runner, there's only 1 test per file, and 1 file per test suite, so both should be marked skipped, no?

**Comment by cpojer** [comment]
@rogeliog what about setting https://github.com/rogeliog/jest-runner-eslint/blob/master/src/utils/toTestResult.js#L18 and https://github.com/rogeliog/jest-runner-eslint/blob/master/src/utils/toTestResult.js#L13 ?
